sources/packages/api/src-sql/RawSQL/FederalRestrictions/Bulk-insert-new-restrictions.sql

```diff
@@ -0,0 +1,26 @@
+/*
+ * Inserts into sims.student_restrictions all the federal restrictions
+ * that are not present and active in the table. The same federal restriction
+ * can be activated and deactivated multiple times for the same student,
+ * generating a new record for every time that the restriction changes its state.
+ */
+INSERT INTO
+    sims.student_restrictions(student_id, restriction_id, is_active)
+SELECT
+    fed_restrictions.student_id student_id,
+    fed_restrictions.restriction_id restriction_id,
+    true is_active

```

Comment:
i didn't get this part @andrewsignori-aot `true is_active`.

Reply:
The result of the select will be inserted, so I gave aliases to the three select columns to match the insert. I though that it would be clear in this way 😉 
![image](https://user-images.githubusercontent.com/61259237/144666039-b9948943-a0b7-4622-b7c5-abe261039ce1.png)